test/simpledynamic.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016-2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdlib.h>              /* For NULL */
+#include <openssl/macros.h>      /* For NON_EMPTY_TRANSLATION_UNIT */
+#include "simpledynamic.h"
+
+#if defined(DSO_DLFCN)
+
+int sd_load(const char *filename, SD *lib)
+{
+    int dl_flags = (RTLD_GLOBAL|RTLD_LAZY);
```

Comment:
Shouldn't this be `RTLD_NOW` for the test to force dependency loading?

Reply:
Ah, `RTLD_LAZY` isn't good for this, good point.

Reply:
Fixed